esignet-service/Dockerfile

```
@@ -1,47 +1,22 @@
-
-# Run as a non-root user for defence-in-depth.
-RUN addgroup -S appgroup && adduser -S appuser -G appgroup
-USER appuser
-
-COPY --from=builder /bin/esignet /bin/esignet
-
-EXPOSE 8088
-
-ENTRYPOINT ["/bin/esignet"]
+FROM debian:bookworm-slim
+RUN apt-get update && apt-get install -y --no-install-recommends ca-certificates openssl && rm -rf /var/lib/apt/lists/*
```

Hadolint (2.14.0)

[warning] Pin versions in apt get install. Instead of `apt-get install <package>` use `apt-get install <package>=<version>`

(DL3008)